src/services/response-service.ts

```diff
@@ -72,45 +75,107 @@ export default class ResponseService extends PercyClientService {
       return
     }
 
-    if (!this.shouldCaptureResource(rootUrl, response.url())) {
-      // Disallow remote redirects.
-      logger.debug(`Skipping [is_remote_redirect] [${width} px]: ${response.url()}`)
-      return
+    if (isRedirect) {
+      // We don't want to follow too deep of a chain
+      if (request.redirectChain().length > 4) {
```

Comment:
I wonder if this even matters now that axios is doing all the work for us..?

Reply:
Looks like the package that axios uses for this has a max set to 21 🤔 https://www.npmjs.com/package/follow-redirects#global-options

I'll update that to match. It would be real interesting to maybe open that to config _if we ever need to_